prooph/pdo-event-store/1.15/post-install.txt

```diff
@@ -0,0 +1,4 @@
+  * Modify your MYSQL_* configuration in <fg=green>.env</>
```

Comment:
Do you support PostgreSQL? If yes, it should be used instead (as this is what we're using by default in recipes.)

Reply:
Yes, Prooph supports PostgreSQL. I'll change that.